catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/MetacardApplication.java

```diff
@@ -214,6 +219,8 @@ public MetacardApplication(
     this.configuration = configuration;
     this.noteUtil = noteUtil;
     this.subjectIdentity = subjectIdentity;
+    this.systemUserList.addAll(
```

Comment:
✏️ How about `Arrays.asList(accessControlSecurityConfiguration.getSystemUserAttributeValue().split(",")));` instead of creating a temporary list just to add it to another

Reply:
If the configuration changes then `systemUserList` won't be updated. I think we'll have to call `getSystemUserAttributeValue()` in `isElevatedUser()` to get the current value.